bigquery_etl/shredder/config.py

```diff
@@ -238,6 +238,12 @@ def fields(self) -> tuple[str, ...]:
     client_id_target(table="telemetry_stable.untrusted_modules_v4"): DESKTOP_SRC,
     client_id_target(table="telemetry_stable.update_v4"): DESKTOP_SRC,
     client_id_target(table="telemetry_stable.voice_v4"): DESKTOP_SRC,
+    client_id_target(table="telemetry_derived.mobile_engagement_clients_v1"): (
+        DeleteSource(table="firefox_ios.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="focus_ios.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="focus_android.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="fenix.deletion_request", field=GLEAN_CLIENT_ID),
+    ),
```

Comment:
I just looked at the code again and remembered the field needs to be repeated for each delete source, like the `messaging_system` ones below this.

```suggestion
    DeleteTarget(
        table="telemetry_derived.mobile_engagement_clients_v1", 
        field=(CLIENT_ID, CLIENT_ID, CLIENT_ID, CLIENT_ID),
    ): (
        DeleteSource(table="firefox_ios.deletion_request", field=GLEAN_CLIENT_ID),
        DeleteSource(table="focus_ios.deletion_request", field=GLEAN_CLIENT_ID),
        DeleteSource(table="focus_android.deletion_request", field=GLEAN_CLIENT_ID),
        DeleteSource(table="fenix.deletion_request", field=GLEAN_CLIENT_ID),
    ),
```

This line is why https://github.com/mozilla/bigquery-etl/blob/1d1addb86cd097f7c3cf03d84ec1c30537591a47/bigquery_etl/shredder/delete.py#L217

Reply:
Updated, thank you for noticing that!